docs/faq.md

```diff
@@ -1,3 +1,20 @@
+## Where are the generated files?
+
+There are 3 places you might see files getting generated. On _every_ build files
+might go to _cache_ (`.dart_tool/build/generated/*`), and _source_ (the working
+tree of your package). These are determined by the `build_to` configuration of
+the Builders you are using. When a Builder specifies `build_to: source` you will
+see them next to your other files but should not edit them by hand, and they are
+generally meant to be published with your package. If you see a warning about
+"conflicting outputs" they refer to _source_ outputs, because when the build
```

Comment:
they _usually_ refer to _source_ outputs

Reply:
When we say "conflicting" we mean source outputs, I think our other error is "duplicate" outputs for when 2 builders want to produce the same asset.